sql/postgres/sqlspec.go

```diff
@@ -290,14 +290,20 @@ func schemaSpec(schem *schema.Schema) (*doc, error) {
 	}
 	d.Schemas = []*sqlspec.Schema{s}
 	d.Tables = tbls
+
+	added := make(map[string]struct{})
+
```

Comment:
```suggestion
	enums := make(map[string]struct{})
```

No need for extra new lines.